subgraphs/maker-governance/protocols/maker-governance/src/ds-chief.ts

```diff
@@ -121,20 +88,20 @@ export function handleEtch(event: Etch): void {
   const to = event.transaction.to;
   // Check if txn is not directly to Chief, it's either to vote delegate or multi-sig + delegate
   if (to && to != event.address) {
-    const fromAdmin = DelegateAdmin.load(sender);
+    const fromAdmin = Voter.load(sender);
     if (!fromAdmin) {
```

Comment:
This looks like it was checking if someone was a delegate owner or not. Now the logic is just checking if they are a voter (from what it seems).

Reply:
yeah, so maybe i should check if the voter is a delegate? I'll try that. 

also i'm not super sure if delegateadmin always was a delegate admin before